settings.gradle

```diff
@@ -40,6 +40,7 @@ pluginManagement {
         id 'com.github.ben-manes.versions' version dependencyCheckVersion
         id "com.jfrog.artifactory" version artifactoryPluginVersion
         id 'io.snyk.gradle.plugin.snykplugin' version snykVersion
+        id "org.cyclonedx.bom" version "1.7.4"
```

Comment:
Can we externalize this property to gradle.properties file in line with others above this